tests-with-coverage/Readme.md

```diff
@@ -0,0 +1,611 @@
+# Measuring the coverage of your tests
```

Comment:
`README.md` is the common naming nomenclature

Reply:
Renamed

Reply:
I still see Readme.md instead of README.md

Reply:
I struggled a bit with git but it should be OK this time